bugbug/bugzilla.py

```diff
@@ -245,14 +300,44 @@ def _find_linked(
 
 
 def find_blocked_by(bug_map: dict[int, BugDict], bug: BugDict) -> list[int]:
+    """Finds all the bugs in the given bug_map that are blocked by the given bug.
+
+    Args:
+        bug_map: A mapping of bug IDs to their corresponding BugDict objects.
+        bug: The bug for which to find all blocked bugs.
+
+    Returns:
+        The IDs of all the bugs in the bug_map that are blocked by the given bug.
+    """
     return _find_linked(bug_map, bug, "blocks")
 
 
 def find_blocking(bug_map: dict[int, BugDict], bug: BugDict) -> list[int]:
+    """Given a dictionary of bugs and a particular bug, return a list of bugs that the particular bug is blocking.
+
+    Args:
+        bug_map: A mapping of bugs with their IDs as keys and their information as values.
+                bug: Information for the bug being queried.
```

Comment:
the indent is not correct

Reply:
this is caused by the black formatting hook, it reformats it like this everytime I run pre-commit

Reply:
no need, fixed it